vevestaX/vevesta.py

```diff
@@ -302,9 +303,33 @@ def __profilingReport(self, fileName):
              }
         ]
         profilingDataframe = pandas.DataFrame(data)
+
+        profileOfVariableDataframe = pandas.DataFrame(
+            {"Field Name": ["Distinct", "Distinct (%)", "Missing", "Missing (%)", "Infinite", "Infinite (%)", "Mean",
+                            "Minimum", "Maximum", "Zeros", "Zeros (%)", "Negative", "Negative (%)",
+                            "Total Memory Size"]})
+
+        for col in self.__data.columns:
+            col_dict = {"Distinct": self.__data[col].nunique(),
+                        "Distinct (%)": self.__data[col].nunique() * 100 / self.__data.shape[0],
+                        "Missing": self.__data[col].isna().sum(),
+                        "Missing (%)": (self.__data[col].isnull().sum() * 100) / (self.__data.shape[0]),
+                        "Infinite": np.isinf(self.__data[col]).values.sum(),
+                        "Infinite (%)": np.isinf(self.__data[col]).values.sum() * 100 / (self.__data.shape[0]),
+                        "Mean": self.__data[col].mean(),
```

Comment:
did u check what will happen if all the values in the column are empty or is none or is na or is np.inf